tests/components/mqtt/test_alarm_control_panel.py

```diff
@@ -746,3 +746,31 @@ async def test_entity_debug_info_message(hass, mqtt_mock):
     await help_test_entity_debug_info_message(
         hass, mqtt_mock, alarm_control_panel.DOMAIN, DEFAULT_CONFIG
     )
+
+
+async def test_state_topic_template_with_entity_id(hass, mqtt_mock):
+    """Test entity id in template."""
+    assert await async_setup_component(
+        hass,
+        alarm_control_panel.DOMAIN,
+        {
+            alarm_control_panel.DOMAIN: {
```

Comment:
MINOR: import DOMAIN.

Reply:
That may be confused with the MQTT `DOMAIN`, i think this is OK as is.